toolkit/tools/pkg/imagecustomizerlib/shrinkfilesystems.go

```diff
@@ -93,16 +102,23 @@ func getStartSectors(imageLoopDevice string, partitionCount int) (matchStarts []
 }
 
 // Get the filesystem size in sectors.
-func getFilesystemSizeInSectors(resize2fsOutput string, imageLoopDevice string) (filesystemSizeInSectors int, err error) {
+func getFilesystemSizeInSectors(resize2fsStdout string, resize2fsStderr string, imageLoopDevice string,
+) (filesystemSizeInSectors int, err error) {
+	if strings.Contains(resize2fsStderr, "Nothing to do!") {
```

Comment:
nit: consider extracting `"Nothing to do!"` into a const?

Reply:
Fixed